_posts/2020-03-30-team-working-agreements.md

```diff
@@ -0,0 +1,315 @@
+reference on the team's preferred methods of working. The working agreement isn't set in stone, and as the team
+changes it's important to update it... but having it in writing helps new members acclimate quickly.
+
+## What are some challenges with team working agreements?
+
+### Uncertainty about what the team agreement should include.
+
+Some teams have struggled to identify what belongs on a team working agreement. Should it describe our team values?
+Habits? Procedures? Rules? Favorite afternoon snacks?
+
+It likely varies from team to team what you want here. Some teams might require guidance around working with JIRA;
+others might be more focused on taking ownership of problems. These differences are likely a reflection of the
+problems the team is currently facing or has recently faced.
+
+It's definitely important that you identify what you're looking for up front, and make it clear heading into the
+brainstorm. Suggest categories, sample habits, & questions to ask heading into the brainstorm, and keep them
+visible during the meeting.
+
+Take notice of known problem areas: are they addressed or avoided in the agreement? It is easy for a team to avoid
+confrontation at their own expense.
+
+Also be cautious about introducing individual bias.
+
+<aside class="dialogue">
+  <div class="answer">
+    <img src="/images/dialogue/steve.jpg">
+    <p class="intro">Steve Hicks says...</p>
+    <p>
+      During my first team agreement brainstorm, I felt like we spent a lot of time talking about details 
+      related to specific incidents. On my next team, I made a strong effort to constrain the suggestions 
+      to avoid this. To me, team values were really important for this new team - and I did such a good 
+      job of encouraging those types of suggestions that it's pretty much all we ended up with.
+    </p>
+    <p>
+      As a result, our initial working agreement lacked important procedural and tactical definition. We've 
+      been adding things like JIRA hygiene and responsibilities of project leaders as they come up, but I would
+      not push a working agreement brainstorm so strongly in one direction again.
+    </p>
+  </div>
+</aside>
+
+Remember that the working agreement is a living document. If the team learns that it missed something in the
+brainstorm, don't hesitate to update the agreement. Team retros are a great time to make updates.
+
+### Working agreements are hard to define for teams without a well-established identity.
+
+For a team that has a strong and cohesive identity, the agreement is likely to affirm many things the team is
+already believing and doing. For a less cohesive team the agreement is harder to pin down. Many opinions will
+surface, and they may be in conflict with each other.
+
+While this is indeed challenging, it is also important to note that teams that lack identity value _greatly_ from
```

Comment:
```suggestion
While this is indeed challenging, it is also important to note that teams that lack identity benefit _greatly_ from
```

I read "identity value" as a compound word instead of reading "value" as a verb, so think that "benefit" would remove that ambiguity.